wordpress-mobile/WordPress-iOS
# Fastlane: Split env files into project.env and user.env so that project.env can be shared

As discussed, this splits the env file used for Fastlane (currently just `~/.wpios-env.default`) into two files: `.configure-files/project.env` and the existing `~/.wpios-env.default`. The latter is copied from the secrets repo and the former should be filled in with personal credentials. Example files for each can be found in `Scripts/fastlane/env`

I have also added a new method, `get_required_env()` which gets a value from `ENV`, but throws an error if it is missing. This ensures that if we add a key to `~/.wpios-env.default`, Fastlane will not continue if that key is missing. Yesterday, @jkmassel and I noticed that some Sentry keys were missing. It should prevent that from happening in future.

To test:

- Run `rake dependencies`. Confirm that `.configure-files/project.env` now exists.
- Ensuring you have an env file at `~/.wpios-env.default`, run any lane (e.g. `bundle exec get_pullrequests_list`). See that there are no errors.

Update release notes:
